test/e2e/encryption_at_rest_test.go

```diff
@@ -146,7 +194,8 @@ func fillKMSforAWS(encAtRest *v1.EncryptionAtRest, atlasAccountArn, assumedRoleA
 	}
 
 	Expect(encAtRest.AwsKms.Region).NotTo(Equal(""))
-	awsAction := cloud.NewAwsAction()
+	awsAction, err := cloud.NewAWSAction(GinkgoT())
+	Expect(err).ToNot(HaveOccurred())
```

Comment:
Was this failing silently before?

Reply:
Never mind, seems it changed signature

Reply:
No, this change was a refactor done in the test utility